shared/control/parallel_dd.control

```diff
@@ -13,15 +15,13 @@ the files system.
 path = glob.glob("/dev/[hsv]d?")
 path.sort()
 fs = path[-1]
-platform = subprocess.check_output("uname -m", shell=True)
```

Comment:
In python2.6, no function "check_output" in subprocess, so replace it by "Popen"